NuGet.Config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <clear />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
+    <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
```

Comment:
Because that's the version that was used in dotnet/sdk repo at the time. Now it's a newer version. I'll update the version in a new PR.

https://cs.github.com/dotnet/sdk/blob/bc5665d1ed4f935766a4449a56cbaae935729365/eng/Versions.props#L45